db/payment_icons.yml

```diff
@@ -1157,4 +1157,40 @@
 -
   name: snap_checkout
   label: Snap Checkout
+  group: other
+
+  name: sveaeramaksu
+  label: Svea Erämaksu
+  group: other
+
+  name: sveapartpayment
+  label: Svea Part Payment
+  group: other
+
+  name: sveadelbetalning
+  label: Svea Delbetalning
+  group: other
+
+  name: svealasku
+  label: Svea Lasku
+  group: other
+
+  name: sveainvoice
+  label: Svea Invoice
+  group: other
+
+  name: sveafaktura
+  label: Svea Faktura
+  group: other
+
+  name: sveayrityslasku
+  label: Svea Yrityslasku
+  group: other
+
+  name: sveab2binvoice
+  label: Svea B2B Invoice
+  group: other
+  
+  name: sveab2bfaktura
+  label: Svea B2B Faktura
   group: other
```

Comment:
Looks like these entries are missing the preceding `-`. Please see other entries for reference. (or see below)

https://github.com/activemerchant/payment_icons/blob/b2ebb1241bc8bb7da0d2a196c7576696d7a0bfe4/db/payment_icons.yml#L1-L4